internal/operators/metallb/config.go

```diff
@@ -0,0 +1,12 @@
+package metallb
+
+type Config struct {
+	MetalLBCPUPerHost          int64  `envconfig:"METALLB_CPU_PER_HOST" default:"1"`
```

Comment:
I don't remember if we will add those values to hw node requirements. I think in case of metal lb we should not require more cpu or memory. Can you check that we don't add those values to hw requirements? or maybe you think we need to add them?

Reply:
@flaper87 what should we do

Reply:
@flaper87 validations? at the moment when I enable metallb-operator it automatically requires additional CPU per node and about 1GB memory per node

Reply:
in GetPreflightRequirements we return these values

Reply:
In that case i think we should set 0 , no reason to add one more core for it and more memory. Storage operators are cpu consuming that's why we add it but metalb should not have those problems i assume

Reply:
@flaper87 @tsorya I removed the hardware requirements. I think it is the right thing to do.